2765. Longest Alternating Subarray/solution.py

```diff
@@ -11,39 +10,81 @@ def alternatingSubarray(self, nums: List[int]) -> int:
 
     	# declare a subArraySize variable that keeps track of length of 
     	# alternating subarrays.
+        subArraySize = 0
 
-    	# declare a list to hold potential subArraySizes.
-
+    	# variable to hold the largest size
+        largestSize = 0
+		# variable to keep track of index in found subarray
+        subArrayIndex = 0
+        
+        # boolean to determine if found sub array in while loop iteration
+        foundSubArray = False
+        
     	# loop through the array starting from 0, increment index by 1 
     	# each time. this loop is the outer loop.
```

Comment:
Why do you have to use 2 nested loops?
If you check my code, you'll see that I only use 1. It takes advantage of the fact that 2 subarrays can only share 1 element. To me, [2, 3, 2, 3] counts as 1 subarray.